### ISSUE TITLE
[Due for payment 2026-01-27] [$250] Keyboard Navigation: Many Pages: The elements are focused in a confusing order

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#76920]
## Action Performed:
Prerequisite:
- the user is logged in

1. Using Windows + Chrome, open the page https://new.expensify.com/settings/profile
2. Navigate through the page using TAB key
3. Observe the focus order

Other occurrences:
1. On Chat flow

## Expected Result:
When a user navigates through the Profile page, it is expected focus would move in a logical order, normally from top left side of the screen, to bottom right side of the screen. In this instance, the left navigation panel, then the middle, then the right

## Actual Result:
As users navigate through the profile page, focus starts in the middle panel (Profile, subscription, wallet, preferences, security, etc. ), before moving onto the left panel (Received, information, spaces, etc. ), then the right panel (Name, contact method, state, pronouns, etc.).

## Area issue was found in:
Settings–Profile

## Failed WCAG checkpoints
WCAG 1.3.2 Meaningful Sequence
https://www.w3.org/WAI/WCAG22/Understanding/meaningful-sequence

## User impact:
A meaningful sequence that matches a logical reading order helps people who rely on assistive technologies that read content aloud. The meaning evident in the sequencing of the information in the default presentation will be the same when the content is presented in spoken form.

## Suggested resolution:
Ensure the focus order matches the visual order by reordering the DOM order.
https://www.w3.org/WAI/WCAG21/Techniques/css/C27

## Environment(s)
Windows Windows 10 - Chrome

## Screenshots/Videos
<img width="1918" height="996" alt="Image" src="https://github.com/user-attachments/assets/fb8e6620-0d73-4b0c-b843-aa36fb60463b" />

https://github.com/user-attachments/assets/da0b87a0-a3a4-4c08-a5dd-fda61411d60a

**Version Number:** NA 
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6994050
**Issue reported by:** NA
**Slack conversation:** NA

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~022009685557512758762</li>
        <li>Upwork Job ID: 2009685557512758762</li>
        <li>Last Price Increase: 2026-01-09</li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @mallenexpensify</details>

**Comment by JmillsExpensify** [comment]
$250 approved for @ShridharGoel

**Comment by twisterdotcom** [comment]
@QichenZhu has submitted payment for this report on ND [here](https://staging.new.expensify.com/search?q=sortBy:date%20sortOrder:desc%20type:expense%20groupBy:reports%20reportID:R00N0cYoVqIG%20policyID:9132779747441AFB).

SO: https://stackoverflowteams.com/c/expensify/questions/7582
- [x] Vendor found in [VML](https://docs.google.com/spreadsheets/d/1Fs-umMmHhCCMzxiczyNydl9Rd6jdO0pQskYA3RvNWQc/edit?gid=1740618654#gid=1740618654): True. Status: Active
- [x] Amount requested: $250.00 (Matches the payment summary here: https://github.com/Expensify/App/issues/76920#issuecomment-3807989698.)
- [x] I, @twisterdotcom am not already the BZ assignee. I have approved this payment.

<details><summary>Zapier Logs</summary>
<b>Run ID:</b> <a href="https://zapier.com/editor/311565481/run/00040eee-11f9-a699-c16a-97fe6d200204/run-details">00040eee-11f9-a699-c16a-97fe6d200204</a>
</details>
